packages/asc-ui/src/components/Tabs/Tabs.test.tsx

```diff
@@ -156,4 +168,47 @@ describe('Tabs', () => {
 
     expect(container.querySelector('#tab-one')).toHaveAttribute('title', 'foo')
   })
+
+  it('should be able to set the active initial Tab by giving an index', () => {
+    const { container } = render(
+      <Tabs label="An example of tabs" initialTabIndex={2}>
+        <Tab id="one" label="First">
+          Contents of the first tab.
+        </Tab>
+        <Tab id="two" label="Second">
+          Contents of the second tab.
+        </Tab>
+        <Tab id="three" label="Third">
+          Contents of the third tab.
+        </Tab>
+      </Tabs>,
+    )
+
+    expect(container.querySelector('#tab-three')).toHaveAttribute(
+      'tabindex',
+      '0',
+    )
+  })
+
+  it('should log a warning and set the active tab to the first when passing a wrong initialTabIndex', () => {
```

Comment:
Why shouldn't the active tab index be the last index when the parameter value exceeds the number of tabs?

Reply:
Well, the default value is the first tab, when the user exceeds then number of tabs it basically means that they just gave the wrong ID, and therefore I thought it would make sense to fallback to the default tab (the first one)